docs/docs-next/src/styles/custom.scss

```diff
@@ -3,273 +3,156 @@
  * bundles Infima by default. Infima is a CSS framework designed to
  * work well for content-centric websites.
  */
-@font-face {
-  font-family: 'Geist';
-  src: url('/fonts/GeistVF.woff') format('woff');
-}
 
-@font-face {
-  font-family: 'Geist-Mono';
```

Comment:
rip geist

Reply:
still there, just refactored this and moved the font-face stuff into `theme-globals.scss`